content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -28,8 +28,8 @@ Init containers are exactly like regular containers, except:
 * Init containers always run to completion.
 * Each init container must complete successfully before the next one starts.
 
-If a Pod's init container fails, Kubernetes repeatedly restarts the Pod until the init container
-succeeds. However, if the Pod has a `restartPolicy` of Never, Kubernetes does not restart the Pod.
+If a Pod's init container fails, the kubelet repeatedly restarts that init container it container succeeds. 
```

Comment:
```suggestion
If a Pod's init container fails, the kubelet repeatedly restarts that init container until it succeeds. 
```

The sentence doesn't make sense. Should it not be this?

Reply:
Right. The sentence you fixed is correct.
I'll commit the suggestion.